src/mpid/ch4/shm/posix/eager/iqueue/iqueue_recv.h

```diff
@@ -0,0 +1,148 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2006 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ *  Portions of this code were written by Intel Corporation.
+ *  Copyright (C) 2011-2017 Intel Corporation.  Intel provides this material
+ *  to Argonne National Laboratory subject to Software Grant and Corporate
+ *  Contributor License Agreement dated February 8, 2012.
+ */
+#ifndef POSIX_EAGER_IQUEUE_RECV_H_INCLUDED
+#define POSIX_EAGER_IQUEUE_RECV_H_INCLUDED
+
+#include "iqueue_impl.h"
+
+/* Work backward through the queue to find the first cell and put that one in the transport's
+ * first_cell position. */
+static void MPIDI_POSIX_EAGER_IQUEUE_enqueue_cell(MPIDI_POSIX_EAGER_IQUEUE_transport_t * transport,
+                                                  MPIDI_POSIX_EAGER_IQUEUE_cell_t * cell)
```

Comment:
Why the `MPIDI_POSIX_EAGER_IQUEUE_enqueue_cell` and `MPIDI_POSIX_EAGER_IQUEUE_receive_cell` functions use uppercase `EAGER_IQUEUE` ? Can we make them lowercase to be consistent with other functions ?

Reply:
Done.